lib/Component.php

```diff
@@ -188,38 +184,38 @@ public function render($params = array()) {
     return $mustache->render($template_html, $template_data);
   }
 
-  static private function recursiveMustache(&$obj, $data) {
-    static $mustache;
-
-    if (!isset($mustache)) $mustache = new \Mustache_Engine;
-
-    switch (gettype($obj)) {
-      case 'string': $obj = $mustache->render($obj, $data); break;
-      case 'object':
-      case 'array':  foreach ($obj as $key => &$value) self::recursiveMustache($value, $data); break;
-    }
-  }
-
-  static private function getJsonrpc($endpoint, $method, $params) {
-    $request_json = json_encode(
-      array(
-        'jsonrpc' => '2.0',
-        'id'      => 1,
-        'method'  => $method,
-        'params'  => $params,
-      )
-    );
-
-    $ch = curl_init();
-    curl_setopt($ch, CURLOPT_URL, $endpoint);
-    curl_setopt($ch, CURLOPT_POST, 1);
-    curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-    curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
-    curl_setopt($ch, CURLOPT_POSTFIELDS, $request_json);
-
-    $response_json = curl_exec($ch);
-    $response = json_decode($response_json);
-
-    return $response->result;
-  }
+  //static private function recursiveMustache(&$obj, $data) {
```

Comment:
Better up - I'll remove them.  A bad habit to leave code commented out when I know it's available in git history when I need it.